src/lib/plugin.js

```diff
@@ -218,40 +218,49 @@ module.exports = class PluginPaymentChannel extends EventEmitter2 {
   }
 
   async connect () {
-    if (!(this._info && this._prefix)) {
-      this.debug('info not available locally, loading remotely')
-      const btpResponse = await this._rpc.message(
-        [{
-          protocolName: 'info',
-          contentType: Btp.MIME_APPLICATION_OCTET_STREAM,
-          data: Buffer.from([ INFO_REQUEST_FULL ])
-        }]
-      )
-      const resp = protocolDataToIlpAndCustom(btpResponse)
-      this._info = (resp.protocolMap && resp.protocolMap.info) || {}
-      this._prefix = this.getInfo().prefix
-    }
+    if (this._connected) return
+    this._connected = true
```

Comment:
shouldn't there be a 'connecting' state, so that `plugin.isConnected()` still returns false at this point?